### PR TITLE
Add more to logrank results object

### PR DESCRIPTION
**Pull request by tavinathanson**
Pass through `Series`s into the results object.

@arahuja

### CODE REVIEW

File: cohorts/survival.py [hunk 1]
```diff
@@ -51,13 +53,18 @@ def plot_kmf(df,
...
     kmf = KaplanMeierFitter()
     if threshold is not None:
-        if threshold == 'median':
+        is_median = threshold == "median"
+        if is_median:
             threshold = df[condition_col].median()
+        label_suffix = "%.2f" % threshold
         condition = df[condition_col] > threshold
-        label = '{} > {:.2f}'.format(condition_col, threshold)
+        label_no_condition = "%s ≤ %s" % (condition_col, label_suffix)
```
Comment (arahuja) [inline cohorts/survival.py:62]: 👍

**Comment by tavinathanson** [comment]
@arahuja also added in the requests for survival curve labeling [here](https://github.com/hammerlab/bladder-analyses/issues/140#issuecomment-240167312) (apologies for linking to a private repo).

File: cohorts/survival.py (hunk 1 above)
Comment (arahuja) [inline cohorts/survival.py:62]: That was for the nice looking `<=`. Am I the only one that prefers `.format` to `%`?

**Comment by arahuja** [comment]
LGTM

File: cohorts/survival.py (hunk 1 above)
Comment (tavinathanson) [inline cohorts/survival.py:62]: I prefer `%` since it's just less chars; I imagine everyone prefers what they're used to?

**Comment by tavinathanson** [comment]
@arahuja per your comment in the repo, I now pass through the whole series through Fisher's: https://github.com/hammerlab/cohorts/pull/121/commits/1bc9714162a5df04b942ecbf48b6f5850e6a2ed4

**Comment by arahuja** [comment]
LGTM

**Comment by tavinathanson** [comment]
@arahuja a few more minor changes, going to merge on 🍏  unless I hear otherwise.

**Pull request merged**